amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -198,7 +199,18 @@ router.get('/:startTimestamp', async (req, res) => {
  *            description: startTimestamp not found
  */
 router.put('/:startTimestamp', async (req, res) => {
-   res.end();
+   let startTimestamp = req.params.startTimestamp;
```

Comment:
startTimestamp doesn't change, change let to const
```suggestion
   const startTimestamp = req.params.startTimestamp;
```